### ISSUE TITLE
Gracefully skip live videos (and placeholders)

### ISSUE THREAD

**Issue by hpsu** [woefe/ytcc#9]
Some youtube channels put up placeholders for future live videos. When such videos are encountered, ytcc fails to download them with:

```
ERROR: This video is unavailable.
An Error occured while downloading the video
```

After which ytcc hard exits. Resulting in preceding videos not being marked as downloaded and locking up the download queue until the offending video is manually marked as watched.

This obviously breaks cron :p

**Comment by hpsu** [comment]
The command used is `ytcc -ud` but can be reproduced by with `ytcc -d <WORKING_ID> <BROKEN_ID>`
And yes, even videos after the "broken" one seem to download but nothing is marked as watched. 

A current example of a live video placeholder https://www.youtube.com/watch?v=IJXVi9Xfa34

## --bug-report-info:
```ini
---ytcc version---
1.7.1

---youtube-dl version---
2018.10.05

---feedparser version---
5.2.1

---lxml version---
4.2.5

---dateutil version---
2.7.3

---python version---
3.7.0 (default, Sep 15 2018, 19:13:07) 
[GCC 8.2.1 20180831]

---mpv version---
mpv 0.29.1 Copyright © 2000-2018 mpv/MPlayer/mplayer2 projects
 built on Fri Oct  5 08:08:09 CEST 2018
ffmpeg library versions:
   libavutil       56.14.100
   libavcodec      58.18.100
   libavformat     58.12.100
   libswscale      5.1.100
   libavfilter     7.16.100
   libswresample   3.1.100
ffmpeg version: n4.0.2


---config dump---
[YTCC]
dbpath = ~/.local/share/ytcc/ytcc.db
downloaddir = /share/Video/Youtube
mpvflags = --really-quiet --ytdl --ytdl-format=bestvideo/best+bestaudio/best

[youtube-dl]
format = bestvideo[height<=?1080]+bestaudio/best
outputtemplate = %(uploader)s/%(title)s.%(ext)s
loglevel = normal
ratelimit = 0
retries = 0
subtitles = en,sv
thumbnail = on

[quickselect]
enabled = yes
ask = yes
alphabet = sdfervghnuiojkl

[TableFormat]
id = on
date = off
channel = on
title = on
url = off
watched = off
```

**Comment by woefe** [comment]
Fixed in 76a4f17.
